packages/wrangler/src/d1/migrations/create.ts

```diff
@@ -40,17 +40,19 @@ export const d1MigrationsCreateCommand = createCommand({
 	},
 	positionalArgs: ["database", "message"],
 	async handler({ database, message }, { config }) {
+		if (!config.configPath) {
+			throw new UserError(
+				"No configuration file found. Create a wrangler.toml or wrangler.json file to define your D1 database."
```

Comment:
We are trying to discourage TOML usage:


```suggestion
				"No configuration file found. Create a wrangler.jsonc to define your D1 database."
```